[Doc] Add more steps to getting started tutorial (#206)
liqotech/liqo: docs/pages/User/GettingStarted/test.md

## Start an Hello World pod

> First, ensure you are operating in your home cluster. Otherwise, set the `KUBECONFIG` variable to the correct value:

Reviewer: ```suggestion
This step requires some to issue some command to Kubernetes from a shell terminal. If your terminal runs in a machine that does not belong to your cluster (e.g., your personal laptop), you have to set the `KUBECONFIG` variable to the correct value:
```
==> I'm not sure the content of the above text is correct. It seems to me that the kubeconfig is needed anyway.

Author: The point of the first statement is to point to the Home API Server and not the foreign cluster we peered with. I try to rephrase it.

nginx-local    1/1     Running   0          76s   10.244.2.214    worker-node-1
nginx-remote   1/1     Running   0          76s   172.16.97.219   liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
```

## Check the pod connectivity

First, let's retrieve the IP address of the `nginx` pods:

```bash
LOCAL_POD_IP=$(kubectl get pod nginx-local -n liqo-demo --template={{.status.podIP}})
REMOTE_POD_IP=$(kubectl get pod nginx-remote -n liqo-demo --template={{.status.podIP}})
echo "Local Pod IP: ${LOCAL_POD_IP} - Remote Pod IP: ${REMOTE_POD_IP}"
```

If you have direct connectivity with the home cluster from your host (e.g. you are running K3s locally),
you can open a browser and directly check the connectivity to both IP addresses.
Similarly, you can also use `curl`:
```
curl ${LOCAL_POD_IP}
curl ${REMOTE_POD_IP}
```

If you do not have direct connectivity, on the other hand, you can fire up a pod and run `curl` from inside:
```
kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${LOCAL_POD_IP}
kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${REMOTE_POD_IP}

> ```

Reviewer: We're missing a conclusion here: what do we expect the user to see? That both output are the same? That you can reach both? Anything else?

## Expose the pod through a Service

The above `hello-world.yaml` manifest tells Kubernetes to create also a service, which is designed to serve traffic to the previously deployed pods.
The service is a traditional [Kubernetes Service](https://kubernetes.io/docs/concepts/services-networking/service/) and can work with Liqo with no modifications.

> Indeed, inspecting the service it is possible to observe that both `nginx` pods are correctly specified as endpoints.

Reviewer: should we say something more? e.g., that the first endpoint belongs to the first cluster, the second belongs to the foreign cluster?

```
kubectl describe service liqo-demo -n liqo-demo

Name:              liqo-demo
Namespace:         liqo-demo
Type:              ClusterIP
IP:                10.105.51.71
Port:              web  80/TCP
TargetPort:        web/TCP
Endpoints:         10.244.2.214:80,172.16.97.219:80
```

### Check the Service connectivity

First, you have to retrieve the IP address of the service:
```
SVC_IP=$(kubectl get service liqo-demo -n liqo-demo --template={{.spec.clusterIP}})
echo "Service IP: ${SVC_IP}"
```

If you have direct connectivity with the home cluster from your host (e.g. you are running K3s locally),
you can open a browser and directly check the connectivity to the service IP address.
Similarly, you can also use `curl`:
```
curl ${SVC_IP}
```

If you do not have direct connectivity, on the other hand, you can fire up a pod and run `curl` from inside:
```
kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl ${SVC_IP}
```

You can also connect to the service through its _service name_, which exploits the Kubernetes DNS service:

```
kubectl run --image=curlimages/curl curl -n default -it --rm --restart=Never -- curl http://liqo-demo.liqo-demo
```

> 

Reviewer: Similarly to the section before, I'm missing the conclusions here.
What do we expect the user to see/to conclude once coming here?

### Check Service connectivity 

> First, you have to scrape the IP address of the service:

Reviewer: ```suggestion
First, you have to retrieve the IP address of the service:
```

```bash
SVC_IP=$(kubectl get svc test-liqo -n test-liqo --template={{.spec.clusterIP}})
echo $SVC_IP
```

If you have direct connectivity with the cluster (e.g. K3s) from your host, open a browser and connect to the value of `$SVC_IP` or use the following `curl` command:

```
curl -v $SVC_IP
```

> If you have not direct connectivity, you can fire up a pod and run a curl inside:

Reviewer: ```suggestion
If you do not have direct connectivity, you can fire up a pod and run a curl inside:
```

```
kubectl run --image=curlimages/curl tester -n test-liqo -ti --rm -- curl -L $SVC_IP
```

> You can also connect to the service by using the service name, using the Kubernetes DNS service:

Reviewer: ```suggestion
You can also connect to the service through its _service name_, which exploits the Kubernetes DNS service:
```

```
kubectl run --image=curlimages/curl tester -n test-liqo -ti --rm -- curl -L http://test-liqo 
```

> 

Reviewer: I would add some commands that show how to deploy the service either locally or remotely, so that the user can play and test that there's no difference in the user experience.
